diff --git a/node_modules/react-native-video/android/src/main/java/com/brentvatne/exoplayer/VideoPlaybackCallback.kt b/node_modules/react-native-video/android/src/main/java/com/brentvatne/exoplayer/VideoPlaybackCallback.kt
index b10dfb5..2674050 100644
--- a/node_modules/react-native-video/android/src/main/java/com/brentvatne/exoplayer/VideoPlaybackCallback.kt
+++ b/node_modules/react-native-video/android/src/main/java/com/brentvatne/exoplayer/VideoPlaybackCallback.kt
@@ -1,7 +1,6 @@
 package com.brentvatne.exoplayer

 import android.os.Bundle
-import androidx.media3.common.Player
 import androidx.media3.session.MediaSession
 import androidx.media3.session.SessionCommand
 import androidx.media3.session.SessionResult
@@ -16,14 +15,10 @@ class VideoPlaybackCallback : MediaSession.Callback {
             return MediaSession.ConnectionResult.AcceptedResultBuilder(session)
                 .setAvailablePlayerCommands(
                     MediaSession.ConnectionResult.DEFAULT_PLAYER_COMMANDS.buildUpon()
-                        .add(Player.COMMAND_SEEK_FORWARD)
-                        .add(Player.COMMAND_SEEK_BACK)
-                        .build()
+                                .build()
                 ).setAvailableSessionCommands(
                     MediaSession.ConnectionResult.DEFAULT_SESSION_COMMANDS.buildUpon()
-                        .add(SessionCommand(COMMAND.SEEK_FORWARD.stringValue, Bundle.EMPTY))
-                        .add(SessionCommand(COMMAND.SEEK_BACKWARD.stringValue, Bundle.EMPTY))
-                        .build()
+                                .build()
                 )
                 .build()
         } catch (e: Exception) {
diff --git a/node_modules/react-native-video/android/src/main/java/com/brentvatne/exoplayer/VideoPlaybackService.kt b/node_modules/react-native-video/android/src/main/java/com/brentvatne/exoplayer/VideoPlaybackService.kt
index 4760ea5..4d5d2c5 100644
--- a/node_modules/react-native-video/android/src/main/java/com/brentvatne/exoplayer/VideoPlaybackService.kt
+++ b/node_modules/react-native-video/android/src/main/java/com/brentvatne/exoplayer/VideoPlaybackService.kt
@@ -30,24 +30,6 @@ class VideoPlaybackService : MediaSessionService() {
     private var binder = PlaybackServiceBinder(this)
     private var sourceActivity: Class<Activity>? = null

-    // Controls for Android 13+ - see buildNotification function
-    private val commandSeekForward = SessionCommand(COMMAND.SEEK_FORWARD.stringValue, Bundle.EMPTY)
-    private val commandSeekBackward = SessionCommand(COMMAND.SEEK_BACKWARD.stringValue, Bundle.EMPTY)
-
-    @SuppressLint("PrivateResource")
-    private val seekForwardBtn = CommandButton.Builder()
-        .setDisplayName("forward")
-        .setSessionCommand(commandSeekForward)
-        .setIconResId(androidx.media3.ui.R.drawable.exo_notification_fastforward)
-        .build()
-
-    @SuppressLint("PrivateResource")
-    private val seekBackwardBtn = CommandButton.Builder()
-        .setDisplayName("backward")
-        .setSessionCommand(commandSeekBackward)
-        .setIconResId(androidx.media3.ui.R.drawable.exo_notification_rewind)
-        .build()
-
     // Player Registry

     fun registerPlayer(player: ExoPlayer, from: Class<Activity>) {
@@ -59,8 +41,7 @@ class VideoPlaybackService : MediaSessionService() {
         val mediaSession = MediaSession.Builder(this, player)
             .setId("RNVideoPlaybackService_" + player.hashCode())
             .setCallback(VideoPlaybackCallback())
-            .setCustomLayout(immutableListOf(seekForwardBtn, seekBackwardBtn))
-            .build()
+                .build()

         mediaSessionsList[player] = mediaSession
         addSession(mediaSession)
